cli/src/pcluster/utils.py

```diff
@@ -118,7 +120,7 @@
 
 def get_region():
     """Get AWS_DEFAULT_REGION from the environment."""
-    return os.environ.get("AWS_DEFAULT_REGION")
+    return os.environ.get("AWS_DEFAULT_REGION") or boto3.session.Session().region_name
```

Comment:
You could write `return boto3.session.Session().region_name`. The `boto3.session...` will cover the environ var

Reply:
Fixed